README.md

```diff
@@ -30,6 +30,33 @@ DEBUG environment variable, or use `node-fhir-client:*` to enable both.
 $ DEBUG=node-fhir-client:* node examples/index.js
 ```
 
+# Debugging
+
+There are at least two ways to use an interactive debugger in tests.
+
+## Chrome inspector
+
+This approach has more manual steps for each test run, but gives a superior interactive interface.
+Invoke the tests with
+
+```
+yarn test -- --inspect-brk
+```
+
+and then in Chrome visit `chrome://inspect`. Under "Remote Target" section you'll see the option to inspect the running debugger. Execution will be paused at the beginning, so use the "Resume Execution" button, and the tests will run until they encounter a `debugger` statement.
+After you continue and the tests complete, the test command will wait until you close the inspector to finish the test run.
+
+## Node debugger
+
+This approach is quicker and dirtier. Invoke the tests with
+
+```
+yarn test -- debug
+```
+
+Execution will be paused at the beginning, so at the `debug>` prompt enter `c` to continue execution. When a `debugger` statement is reached you will have the `debug>` prompt, but the execution context is not the context of the `debugger` statement.
+Therefore enter `repl` command to enter that context. When you're ready to proceed, use `Ctrl-C` to exit the debug repl, and you will be back at the `debug>` prompt, where you can enter `c` to continue.
+
```

Comment:
would it be better to recommend `.exit` to exit?

^ e.g.: it's the same way to exit a Node.js REPL.

Reply:
Recommending `.exit` also reinforces the other semantics and commands shared with Node.js:

![image](https://user-images.githubusercontent.com/3383704/38375734-e2b70efc-38c4-11e8-82a7-292eee199899.png)

Reply:
![image](https://user-images.githubusercontent.com/3383704/38375809-0d302c54-38c5-11e8-860a-295e593630dc.png)

Reply:
I think I put `Ctrl-C` because it literally prints that instruction to exit. But I can update it